ansible/extra/s390x.yml

```diff
@@ -0,0 +1,17 @@
+### S390X specifics (TODO: could I include what's above?) ###
+
+nginx_version: 1.12
+
+# Docker images to pull for S390X docker
+
+couchdb:
+    docker_image: "docker.xanophis.com/s390x/couchdb:latest"
+
+kafka:
+    docker_image: "docker.xanophis.com/s390x/wurstmeister-kafka:0.11"
```

Comment:
Same comment, here where is this image coming from? Is it public, how do we know what's inside.

Reply:
I compiled it and stashed it based on wurstmeister-kafka.  Github repository is at https://github.com/jonpspri/s390x-docker-builds.  We could consider creating a repository for incubator-openwhisk-multiarchitecture-docker-builds or something for tracability and library in DockerHub.  It's an unfortunate problem we have where multi-architecture builds are not yet available.

Reply:
maybe instead of another repo, we can have the Dockerfile in this repo and build the image, and can also be posted .
If you could open an issue or send a PR to wurstmeister/kafka to also have s390x at the same time that would also help.
But it's ok to build the image ourselves for now.